src/inmanta/data/__init__.py

```diff
@@ -3346,6 +3348,9 @@ class Compile(BaseDocument):
 
     compile_data: Optional[JsonType] = None
 
+    partial: bool = False
+    removed_resource_sets: Optional[List[str]] = None
```

Comment:
I think you will also need to add these fields to the corresponding class in `inmanta.data.model`, and potentially update the `to_dao` and/or `to_dto` methods.